site/it/about/bib.md

```diff
@@ -0,0 +1,131 @@
+# White Papers su TensorFlow
+
+Questo documento identifica i white papers su TensorFlow.
```

Comment:
If you want still maintain It literal It Is better `individua`

Reply:
White paper in the non translation list

Reply:
Check also paper(s) and other not translating the terms https://accademiadellacrusca.it/it/consulenza/plurale-dei-forestierismi-non-adattati/8